Templates/project_collection/README.md

```diff
@@ -0,0 +1,212 @@
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually there is only one species associated with a data collection
+  * Data collections about human subjects should be separate from other species
+  
+#### **is data of type** - *modality*
+  * This is the name of the data modality for this collection
+  * This field is controlled by the vocabulary in the `modality` column on `values` sheet
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually there is only one modality associated with a data collection
+  
+#### **is subject of resource** - *URL*
+  * This is usually a link to the repository where the raw data is stored.
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+
+#### **was collected using** - *technique*
+  * This is the name technique used to acquire data for this collection
+  * This field is controlled by the vocabulary in the `technique` column on `values` sheet
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually there is only one technique associated with a data collection
+  
+#### **Title** - *string*
+  * This is a simple string is currently not displayed. 
+    (note: Title _is_ displayed for _data collection projects_)
+    
+#### **Short title** - *string*
+  * This is a simple string that will be displayed on 
+    https://knowledge.brain-map.org/ as your data collection title
+    (note: Short title _is not_ displayed for _data collection projects_)
+  * Please limit short titles to 30 characters or less
+  
+#### **Description** - *string*
+  * This is a simple string that will be displayed on 
+    https://knowledge.brain-map.org/ as your data collection description
+  
+#### **Access control** - *access control*
+  * This indicates the level of control of access to these data
+  * This field is controlled by the vocabulary in the `access control` column on `values` sheet
+  * Most non-human data collections should be `open`
+  * Human data collections should be `controlled`
+
+#### **Completion state** - *completion state*
+  * This indicates whether or not new data is still being added to this collection
+  * This field is controlled by the vocabulary in the `completion state` column on `values` sheet
+  * If more data are expected to be added to this collection, then the state is `in progress`
+  * If data not more data will be added to this collection, then the state is `complete`
+  
+#### **Last updated date** - *yyyy/mm/dd*
+  * This is the day of the most recent data submission deadline where the `Reported count` was updated
+  * Please input the date in `yyyy/mm/dd` format if possible (some software likes to auto-change this)
+  
+#### **Reported count** - *int*
```

Comment:
This is a comment about the tempalte: I am hesitant to include reported count in this template. I think we should aggregate this from the specimen inventory/sample inventory submission. Can we eliminate this from the form? Or, we can keep here and consider that this is the first estimate and then we should compare to the submitted inventory and edit later?

Reply:
This field could be removed as long as the information is provided to us in some form.

Reply:
Deleted this from template and description.